packages/common/src/search/_internal/commonHelpers/getEventsApiDefinition.ts

```diff
@@ -0,0 +1,15 @@
+import { IApiDefinition } from "../../types/types";
```

Comment:
given what's in this file, I think we could just in-line it into `getApi(...)` function